styles.css

```diff
@@ -58,20 +65,625 @@ ul li a {
+}
+
+.fs {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  padding: 24px;
+  gap: 15px;
+}
+
+.app {
+  font-weight: 400;
+  line-height: 28px;
+  font-size: 20px;
+  font-style: normal;
+  color: #ebebff;
+  text-align: center;
+  left: 24px;
+  margin-top: 15px;
+}
+
+.lorem {
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  margin-top: 40px;
+  gap: 20px;
+}
+
+.lor {
+  display: flex;
+  left: 0;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  align-items: center;
+  margin-top: 20px;
+  padding: 15px;
+  background: #fff;
+}
+
+.border {
+  width: 134px;
+  height: 5px;
+  margin-top: 45px;
+  background-color: #0d151e;
+  border-radius: 10px;
+}
+
+.item {
+  width: 327px;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding-left: 16px;
+  padding-right: 16px;
+  margin-bottom: 60px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.item img {
+  width: 295px;
+  height: 220px;
+  border-radius: 8px;
+  margin-bottom: 12px;
+}
+
+.item h2 {
+  color: #172b4d;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  margin-bottom: 20px;
 }
 
-ul li {
+.item ul {
+  display: flex;
   list-style: none;
+  gap: 12px;
+}
+
+.item ul .canopy {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.diskA {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  margin-bottom: 16px;
+}
+
+.diskA::before {
+  display: inline-block;
+  width: 8px;
+  height: 8px;
+  background: #c1c7d0;
+  content: '';
+  border-radius: 4px;
+  margin-right: 12px;
+}
+
+.item p {
+  display: block;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-bottom: 16px;
+}
+
+.bott {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  border: 1px solid #396df2;
+  border-radius: 8px;
+  cursor: pointer;
+  padding: 12px;
+}
+
+.bott:hover {
+  background: #6070ff;
+  color: #fff;
+  box-shadow: 10px 10px;
+}
+
+.boot:active {
+  background: #6070ff;
+  color: #fff;
+  box-shadow: inset 5px 5px 5px rgba(0, 0, 0, 0.5);
+}
+
+.form {
+  margin-top: 5rem;
+  padding: 6% -1.5rem;
+  color: #fff;
+  background-image: url("asset/background\ img.png");
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  background-size: cover;
+  overflow: hidden;
+  background-clip: inherit;
+  background-position: 1.875rem -3.1875rem;
+  border-radius: 5rem 0 0 0;
+  position: relative;
+}
```

Comment:
- [ ] Ouch! this is very close but it seems like the background image in your design is a bit misaligned as some of the icons in the background image are not in position as shown in the Figma design. As a frontend developer, It's very important you pay attention to small details like this as they give your UI a more polished feel. 😉
- Your design:
  ![image](https://github.com/Surafels/Portfolio_Mobile/assets/104968236/a67935da-9f75-4a52-9a7d-6ec6b6fe64d2)
- Figma design:
  ![image](https://github.com/Surafels/Portfolio_Mobile/assets/104968236/89afdcad-61b9-48de-9cf9-8a524ce3bf65)